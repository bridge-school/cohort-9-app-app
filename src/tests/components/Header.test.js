import React from "react";
import renderer from "react-test-renderer";
import Header from "../../components/Header/Header";

describe("Header", () => {
  test("Should render properly as an admin", () => {
    const tree = renderer
      .create(<Header isAdmin />)
      .toJSON();
    
    expect(tree).toMatchSnapshot();
  });

  test("Should render properly", () => {
    const tree = renderer
      .create(<Header />)
      .toJSON();
    
    expect(tree).toMatchSnapshot();
  });
});
