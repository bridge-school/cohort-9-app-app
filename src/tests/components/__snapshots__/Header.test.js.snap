// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render properly 1`] = `
<nav
  className="sc-bdVaJa gbvVgg"
>
  <div
    className="ui two column grid"
  >
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="right aligned column"
      >
        <div
          className="sc-bwzfXH ixKSS"
        >
          Admin view
        </div>
      </div>
    </div>
  </div>
</nav>
`;

exports[`Header Should render properly as an admin 1`] = `
<nav
  className="sc-bdVaJa gbvVgg"
>
  <div
    className="ui two column grid"
  >
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="right aligned column"
      >
        <div
          className="sc-bwzfXH ixKSS"
        >
          Admin view
        </div>
      </div>
    </div>
  </div>
</nav>
`;
